IdentityCore/src/oauth2/account/MSIDAccount.m

```diff
@@ -173,7 +173,7 @@ - (BOOL)isHomeTenantAccount
 
 - (NSString *)description
 {
-    return [NSString stringWithFormat:@"MSIDAccount environment: %@ storage environment %@ realm: %@ username: %@ homeAccountId: %@ accountType: %@ localAccountId: %@", self.environment, self.storageEnvironment,  self.realm, self.username, self.accountIdentifier.homeAccountId, [MSIDAccountTypeHelpers accountTypeAsString:self.accountType], self.localAccountId];
+    return [NSString stringWithFormat:@"MSIDAccount environment: %@ storage environment %@ realm: %@ username: %@ homeAccountId: %@ accountType: %@ localAccountId: %@", self.environment, self.storageEnvironment,  self.realm, MSID_PII_LOG_EMAIL(self.username), self.accountIdentifier.homeAccountId, [MSIDAccountTypeHelpers accountTypeAsString:self.accountType], self.localAccountId];
```

Comment:
Looks like in other changes above, homeAccountId was tagged with MSID_PII_LOG_TRACKABLE or maskedHomeAccountId. Do we need to do that here?

Reply:
Addressed